### PR TITLE
Sign In and Up

### LINKED ISSUE #21: User Sign Up Page
- [ ] As a user I should be able to navigate to a sign up page
- [ ] I should be able to create an account with a a username, first name, last name, billing address, payment info, and profile image

### PR DESCRIPTION
**Pull request by nathanielmillard**
## Pull request type
Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other (please describe):

## Problem/feature
- closes #21 
- closes #21 

## Solution
- built out a simple sign in page
- build out a simple sign up page
- build out a user profile page option that aniticipate if a user is signed in
- build out dynamic routes to handle intuitive navigation

## Checklist
- [x] Code compiles correctly
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] All tests passing
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: src/components/ProfilePage/ProfilePage.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import Profile from '../Profile/Profile.js'
+import {Link} from 'react-router-dom'
+
+const ProfilePage = (props) => {
+  const switchBoard = (
+      <section>
+        <h1>Looks like you aren't sign in yet!</h1>
```
Comment (coreycartercodes) [inline src/components/ProfilePage/ProfilePage.js:8]: *Signed in yet

File: src/components/SignIn/SignInPage.js [hunk 2]
```diff
@@ -1,25 +1,45 @@
...
-      <footer className="SignInPage-footer">
-        <Link to='/'><button> Take Me Back </button></Link>
-      </footer>
-    </div>
-  );
+class SignInPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
```
Comment (coreycartercodes) [inline src/components/SignIn/SignInPage.js:9]: Just a question for my own learning. Is this where you could put a default value for the field?

**Review by DougieDev — APPROVED** [review]

File: src/components/ProfilePage/ProfilePage.js (hunk 1 above)
Comment (nathanielmillard) [reply src/components/ProfilePage/ProfilePage.js:8]: Done!

File: src/components/SignIn/SignInPage.js (hunk 2 above)
Comment (nathanielmillard) [reply src/components/SignIn/SignInPage.js:9]: This is actually temporary data storage that is currently being mutated by a function when the user fills out their form.
So this isn;t data a user actually sees, it is logic handling for the ux elements to interact with.

**Pull request merged**